components/PoolForm.tsx

```diff
@@ -89,11 +99,15 @@ const PoolForm: FC<IntrinsicElements["form"] & PoolFormProps> = ({
 
 			try {
 				setTxPending();
-				const tx = await signAndSendTx(
-					extrinsic,
-					selectedAccount.address,
-					wallet.signer
-				);
+				let tx: CENNZTransaction;
+				if (selectedWallet === "CENNZnet")
+					tx = await signAndSendTx(
+						extrinsic,
+						CENNZAccount.address,
+						wallet.signer
+					);
+				if (selectedWallet === "MetaMask")
```

Comment:
should this be else if ?
and can we support 'wallet connect' as well? the sign method used is also supported by that wallet.. We might just want to open a new ticket to see if we can also support that.. in that case we can change the following mtd to 
signViaEthWallet? I plan to release api version today... so you can use this mtd from CENNZnet API directly..

Reply:
renamed to `signViaEthWallet` - I will look at using 'wallet connect' in another PR